values/demo.yaml

```diff
@@ -157,3 +157,11 @@ abha-verification:
   metadata:
     labels:
       environment: demo
+
+bahmni-mart:
+  enabled: true
+  config:
+    CRON_TIME: "*/10 * * * *"
+    MART_DB_NAME: martdb
+  secrets:
+    MART_DB_HOST: "bahmni-dev-postgresql"
```

Comment:
DB name is wrong. It should have `demo` instead of `dev`

Reply:
fixed now. Thanks !!